### PR TITLE
Cleanup old tld refs when switching tld or uninstalling

### PR DESCRIPTION
**Pull request by drbyte**
Previously we would leave old tld stubs around; while it's harmless, it's less tidy.

### CODE REVIEW

File: cli/Valet/DnsMasq.php [hunk 1]
```diff
@@ -145,6 +148,7 @@ function createTldResolver($tld)
     function updateTld($oldTld, $newTld)
     {
         $this->files->unlink($this->resolverPath.'/'.$oldTld);
+        $this->files->unlink(VALET_HOME_PATH . '/dnsmasq.d/tld-' . $oldTld);
```
Comment (mattstauffer) [inline cli/Valet/DnsMasq.php]: Should this instead be the same as what we're using to create it?

```php
        $tldConfigFile = $this->dnsmasqUserConfigDir() . 'tld-' . $tld . '.conf';
```

https://github.com/laravel/valet/blob/master/cli/Valet/DnsMasq.php#L120

I'll be honest, though, this is all a bit magical for me.

**Comment by drbyte** [comment]
Tech background:
Dnsmasq requires two configurations for each TLD:
- a dnsmasq conf file (previously we just dropped this in a generic file; now we do it in separate tld-xxxx.conf files), which the dnsmasq app uses to offer DNS
- a resolver file for the OS to proxy DNS lookups into dnsmasq.

We've been properly cleaning up old resolver files when changing TLD, but not the dnsmasq conf file. This PR brings back that consistency.

File: cli/Valet/DnsMasq.php (hunk 1 above)
Comment (drbyte) [reply cli/Valet/DnsMasq.php]: Agreed - consistency here will make maintenance less complicated.

**Comment by mattstauffer** [comment]
Thanks as always @drbyte!

**Pull request merged**